lib/cancan/ability.rb

```diff
@@ -302,6 +302,9 @@ def extract_subjects(subject)
 
     def alternative_subjects(subject)
       subject = subject.class unless subject.is_a?(Module)
+      unless defined? ActiveRecord
+        return [:all, *subject.ancestors, subject.class.to_s]
+      end
       if subject.respond_to?(:subclasses) && subject < ActiveRecord::Base
```

Comment:
can't we change this if to `subject.respond_to?(:subclasses) && defined?(ActiveRecord::Base) && subject < ActiveRecord::Base` and avoid the code above?